releases.go

```diff
@@ -0,0 +1,37 @@
+package versionbundle
+
+import "sort"
+
+// CanonicalizeReleases canonicalizes representation of multiple releases. It
+// makes sure there are no duplicate versions.
+func CanonicalizeReleases(releases []Release) []Release {
+	releases = deduplicateReleases(releases)
+	// TODO: Add changelog deduplication here.
+
+	return releases
+}
+
+func deduplicateReleases(releases []Release) []Release {
+	if len(releases) < 2 {
+		return releases
+	}
+
+	sort.Sort(SortReleasesByTimestamp(releases))
+	sort.Stable(SortReleasesByVersion(releases))
+
+	for i := 0; i < len(releases)-1; i++ {
+		if releases[i].Version() == releases[i+1].Version() {
+			// NOTE: This will remove newer version of two duplicates to
+			// maintain active version available.
```

Comment:
:exclamation: Please, pay attention here. It his totally possible (and present situation within our ecosystem) to end up in situation where latest release is not active but one of the older ones is. Therefore latest release must not always win in deduplication.